SingularityService/src/main/java/com/hubspot/singularity/data/history/PostgresHistoryJDBI.java

```diff
@@ -253,5 +241,10 @@ void setDeployJson(
     @Bind("json") @Json SingularityDeployHistory deployHistory
   );
 
+  @Override
+  default String getRequestHistoryBaseQuery() {
```

Comment:
If this is just a constant let's declare it as a string on the class, not as a method, same for the mysql jdbi version

Reply:
The user column is named `user` in the mysql version, and `f_user` in the postgres version. That's why I didn't declare it as a constant.

Reply:
ah, totally read over that difference. It can still just be a `String REQUEST_HISTORY_BASE_QUERY = "...";` on each class though, vs a method

Reply:
sry, read that again, ignore me...